src/endpoint/invites-schemaApp.ts

```diff
@@ -25,14 +25,11 @@ export type SchemaAppInvitation = SchemaAppId & {
 	description?: string
 	expiration?: number
 	acceptUrl?: string
+	acceptances: number
```

Comment:
Just want to double-check that this is meant to be a mandatory property, like `id`, rather than optional like all the others.

Reply:
It's not officially in the docs but since it's always there I think we can assume required.